duffel_api/models/airline.py

```diff
@@ -11,6 +11,8 @@ class Airline:
     id: str
     name: str
     iata_code: Optional[str]
+    logo_lockup_url: Optional[str]
```

Comment:
**test**: Could you please add examples of these to the test fixture? [1][2]

[1] https://github.com/duffelhq/duffel-api-python/blob/3f01a35833a40a87b242d5328890491531ffcc04/tests/fixtures/get-airlines.json#L3-L7
[2]
```
    {
      "iata_code": "BA",
      "id": "aln_00001876aqC8c5umZmrRds",
      "name": "British Airways",
      "logo_lockup_url": "https://assets.duffel.com/img/airlines/for-light-background/full-color-lockup/BA.svg",
      "logo_symbol_url": "https://assets.duffel.com/img/airlines/for-light-background/full-color-logo/BA.svg"
    }
```